### PR TITLE
Sets up header

### LINKED ISSUE #15: [Frontend] - Set up mock header

### PR DESCRIPTION
**Pull request by ayoadekunle**
https://photos.google.com/photo/AF1QipMd5dt_PfQLI9RSdTlYhK9X8VuVwVrowZXcSYEH

### CODE REVIEW

File: client/src/Header/header.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import playLogo from './Images/play_logo.png';
+import './header.style.css';
+
+class Header extends Component {
+    render(){
```
Comment (ajwang0) [inline client/src/Header/header.js]: Add space after ()

`render() {`

File: client/src/Header/header.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                <ul className="nav-bar">
+                    <li className="nav-element" id="logo">
+                        <a href="https://play.google.com/store">
+                            <img src={playLogo} alt="Google Play Logo"/>
+                        </a>
+                    </li>
+                    <li className="nav-element" id="link">
+                        <a href="https://bit.ly/30xkTrQ">
+                            Games
+                            </a>
```
Comment (ajwang0) [inline client/src/Header/header.js]: Can you fix the indentation

File: client/src/Header/header.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+                    </li>
+                    <li className="nav-element" id="active">
+                        <a href="https://play.google.com/store/apps">
+                            Apps
+                        </a>
+                    </li>
+                    <li className="nav-element" id="link">
```
Comment (ajwang0) [inline client/src/Header/header.js]: That id isn't unique. Maybe use class instead?

File: client/src/Header/header.style.css [hunk 4]
```diff
@@ -0,0 +1,41 @@
+.nav-bar {
+    list-style-type: none; 
+    margin: 30px;
+    overflow: hidden;
+    width: 100%;
+    z-index: 9999;
+}
+
+.nav-bar li {
+    float: left;
```
Comment (ajwang0) [inline client/src/Header/header.style.css]: Isn't it on the left side by default? Why did we need to use float left?

**Review by ajwang0 — CHANGES_REQUESTED** [review]

**Review by ajwang0 — CHANGES_REQUESTED** [review]
Can you also upload a screenshot of the header to this commit?

File: client/src/Header/header.style.css (hunk 4 above)
Comment (ayoadekunle) [reply client/src/Header/header.style.css]: It makes the navigation elements stay side by side

File: client/src/Header/header.js [hunk 5]
```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import playLogo from './Images/play_logo.png';
+import './header.style.css';
+
+class Header extends Component {
+    render() {
+        return (
+            <div className="header">
+                <ul className="nav-bar">
+                    <li className="nav-element" id="logo">
```
Comment (tonymai) [inline client/src/Header/header.js]: let's avoid ids unless needed by javascript. it's good practice to use class names only for styling

File: client/src/Header/header.js [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+                        <a href="https://play.google.com/store">
+                            <img src={playLogo} alt="Google Play Logo"/>
+                        </a>
+                    </li>
+                    <li className="nav-element link">
+                        <a href="https://bit.ly/30xkTrQ">
```
Comment (tonymai) [inline client/src/Header/header.js]: direct link please. and also, lets link to the game category https://play.google.com/store/apps/category/GAME

File: client/src/Header/header.js [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+                        </a>
+                    </li>
+                    <li className="nav-element link">
+                        <a href="https://play.google.com/store/books">
+                            Books
+                        </a>
+                    </li>
+                    <li className="nav-element link">
+                        <a href="https://play.google.com/store/search?q=kids">
+                            Kids
```
Comment (tonymai) [inline client/src/Header/header.js:37]: Lets link to the category https://play.google.com/store/apps/category/FAMILY

File: client/src/Header/header.style.css (hunk 4 above)
Comment (tonymai) [reply client/src/Header/header.style.css]: let's avoid floats and absolute position for styling. there are other ways to position the tabs, right?

File: client/src/Header/header.style.css [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+}
+
+.nav-bar a {
+    display: block;
+    margin-right: 20px;
+    margin-left: 20px;
```
Comment (tonymai) [inline client/src/Header/header.style.css]: let's use the shorthand, `margin: 0 20px`

File: client/src/Header/header.style.css [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+.nav-bar .link a {
+    color: #7A7A7A;
+    text-decoration: none;
+    border-radius: 4px;
+    border-bottom: 4px solid transparent;
+    transition: 0.3s;
+    padding-left: 4px;
+    padding-right: 4px;
+    padding-top: 18px;
+    padding-bottom: 18px;
```
Comment (tonymai) [inline client/src/Header/header.style.css]: shorthand: `padding: 18px 4px`

File: client/src/Header/header.style.css [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+    color: #525252;
+}
+
+.nav-bar #active a {
+    color: #429E73;
+    border-bottom: 3px solid #429E73;
+    padding-left: 4px;
+    padding-right: 4px;
+    padding-top: 18px;
+    padding-bottom: 18px;
```
Comment (tonymai) [inline client/src/Header/header.style.css]: shorthand

**Review by tonymai — CHANGES_REQUESTED** [review]

File: client/src/Header/header.style.css (hunk 4 above)
Comment (ajwang0) [reply client/src/Header/header.style.css]: +1 look into using flexbox for this.

File: client/src/Header/header.style.css (hunk 9 above)
Comment (ajwang0) [inline client/src/Header/header.style.css]: You can write this as 

`padding: 18px 4px;`

File: client/src/Header/header.style.css (hunk 10 above)
Comment (ajwang0) [inline client/src/Header/header.style.css]: Same here

**Review by ajwang0 — CHANGES_REQUESTED** [review]

File: client/src/App.js [hunk 11]
```diff
@@ -26,7 +27,10 @@ class App extends Component {
     }
     return (
       <div className='App'>
+        <Header />
         <AppOfTheWeek />
+        <StandardCluster />
+        <TopCharts />
```
Comment (ajwang0) [inline client/src/App.js]: Can you remove these two. That's prob why the tests are failing

**Review by ajwang0 — CHANGES_REQUESTED** [review]

**Review by tonymai — DISMISSED** [review]

**Review by ajwang0 — APPROVED** [review]

**Pull request merged**